kafka/integration_test.go

```diff
@@ -1420,6 +1420,7 @@ func (its *IntegrationTestSuite) TestAdminClient_DescribeTopics() {
 		"Expected correct error for nonexistent topic")
 
 	topicDesc := topicDescs[0]
+	assert.NotNil(topicDesc.TopicId)
```

Comment:
Since the type of topicDesc.TopicId is Uuid, it's a struct and not a nullable type, so this condition will always be true. 
It might be better to add a check like, assert.NotZero(topicDesc.TopicId.GetMostSignificantBytes())  and similar for least significant bytes.
Or that assert.NotEmpty(topicDesc.TopicId.String())
(I'm just taking a guess here, would this condition be true?)

Or if it depends on the broker version, and we can't always guarantee that this will be present, then we should omit this altogether, since this check cannot be false.

Reply:
I added this check just to make sure that we are not removing TopicId from the code by mistake. I think we should keep it here.